R/util.R

```diff
@@ -138,15 +138,11 @@ dir_create <- function(x) {
 }
 
 hash_files <- function(filenames, named = TRUE) {
-  if (is.null(filenames)) {
```

Comment:
simplified logic here as we do the check for empty file list elsewhere, and this is consistent with the file size check